docs/rules/no-template-key.md

```diff
@@ -0,0 +1,34 @@
+# Disallow 'key' attribute on '<template>' (no-template-key)
+
+Vue.js disallows `key` attribute on `<template>` elements.
+
+## 📖 Rule Details
+
+This rule reports the `<template>` elements which have `key` attribute.
+
+👎 Examples of **incorrect** code for this rule:
```

Comment:
In my custom, I don't use the notation since it doesn't work outside of GitHub (i.g. jsdoc, editor previews, etc...). Should I use it?